charts/seed-controlplane/charts/kube-apiserver/values.yaml

```diff
@@ -1,100 +0,0 @@
-replicas: 1
-kubernetesVersion: 1.15.2
-priorityClassName: foo
-securePort: 443
-shootNetworks:
-  services: 10.0.1.0/24
-podAnnotations: {}
-featureGates: {}
-  # CustomResourceValidation: true
-  # RotateKubeletServerCertificate: false
-runtimeConfig: {}
-  # autoscaling/v2alpha1: true
-
-oidcConfig: {}
-  # caBundle: |
-  #   -----BEGIN CERTIFICATE-----
-  #   ...
-  #   -----END CERTIFICATE-----
-  # clientID: clientid
-  # groupsClaim: groups
-  # groupsPrefix: prefix
-  # issuerURL: http://localhost
-  # signingAlgs:
-  # - foo
-  # requiredClaims:
-  #   foo: bar
-  #   bar: foo
-  # usernameClaim: user
-  # usernamePrefix: prefix
-
-admissionPlugins:
-- name: Priority
-- name: NamespaceLifecycle
-- name: LimitRanger
-- name: PodSecurityPolicy
-- name: ServiceAccount
-- name: NodeRestriction
-- name: DefaultStorageClass
-- name: PersistentVolumeLabel
-- name: DefaultTolerationSeconds
-- name: StorageObjectInUseProtection
-- name: MutatingAdmissionWebhook
-- name: ValidatingAdmissionWebhook
-- name: ResourceQuota
```

Comment:
This is a very nice catch! Though, we don't need to handle it here since the `PersistentVolumeLabel` is injected by the AWS extension in case CSI is disabled, see https://github.com/gardener/gardener-extension-provider-aws/blob/a752b0306605e98cb6c4297e38c6503591c077fb/pkg/webhook/controlplane/ensurer.go#L179-L182.